### ISSUE TITLE
setFreeJoin declared twice in CreateMeetingParameters

### ISSUE THREAD

**Issue by Magik-72** [bigbluebutton/bigbluebutton-api-php#137]
Since this release https://github.com/bigbluebutton/bigbluebutton-api-php/releases/tag/2.0.10, there is an error introduced by this commit https://github.com/bigbluebutton/bigbluebutton-api-php/commit/890eacad9b0224451108a05b6484041a5274050a.

A second `setFreeJoin` method that conflicts with an already present `setFreeJoin` method in the CreateMeetingParameters class https://github.com/bigbluebutton/bigbluebutton-api-php/blob/master/src/Parameters/CreateMeetingParameters.php.

```php
public function setFreeJoin($freeJoin)
    {
        $this->freeJoin = $freeJoin;

        return $this;
    }
```

and

```php
public function setFreeJoin($guestPolicy)
    {
        $this->guestPolicy = $guestPolicy;

        return $this;
    }
```

This produces this error: 

>  Cannot redeclare BigBlueButton\Parameters\CreateMeetingParameters::setFreeJoin()

Since i'm not familiar with this package I'll not make a PR but I'm confident it should be pretty easy to fix :)

**Comment by tanero** [comment]
Today after upgrade the library by composer,  start to recieve same error.

**Comment by senatoregiorgio** [comment]
I've just encountered the same error. I had to downgrade the library to version 2.0.9.

**Comment by GhaziTriki** [comment]
Fixed in 2.0.11
